cmd/scollector/main.go

```diff
@@ -71,6 +73,15 @@ func main() {
 	} else if conf.Tags["host"] != "" {
 		slog.Fatalf("host not supported in custom tags, use Hostname instead")
 	}
+	if conf.PProf != "" {
+		go func() {
+			slog.Infof("Starting pprof at http://%s/debug/pprof/", conf.PProf)
+			err := http.ListenAndServe(conf.PProf, nil)
```

Comment:
`slog.Fatal(http.ListenAndServe())` is how I most often see this. If it ever returns something bad happened.